impeller/entity/contents/filters/gaussian_blur_filter_contents_unittests.cc

```diff
@@ -508,27 +508,24 @@ TEST(GaussianBlurFilterContentsTest, LerpHackKernelSamplesSimple) {
           },
   };
 
-  GaussianBlurPipeline::FragmentShader::KernelSamples fast_kernel_samples =
+  GaussianBlurPipeline::FragmentShader::BlurInfo blur_info =
       LerpHackKernelSamples(kernel_samples);
-  EXPECT_EQ(fast_kernel_samples.sample_count, 3);
+  EXPECT_EQ(blur_info.sample_count, 3);
 
-  GaussianBlurPipeline::FragmentShader::KernelSample* samples =
-      kernel_samples.samples;
-  GaussianBlurPipeline::FragmentShader::KernelSample* fast_samples =
-      fast_kernel_samples.samples;
+  KernelSample* samples = kernel_samples.samples;
 
   //////////////////////////////////////////////////////////////////////////////
   // Check output kernel.
 
-  EXPECT_FLOAT_EQ(fast_samples[0].uv_offset.x, -1.3333333);
-  EXPECT_FLOAT_EQ(fast_samples[0].uv_offset.y, 0);
-  EXPECT_FLOAT_EQ(fast_samples[0].coefficient, 0.3);
-  EXPECT_FLOAT_EQ(fast_samples[1].uv_offset.x, 0);
-  EXPECT_FLOAT_EQ(fast_samples[1].uv_offset.y, 0);
-  EXPECT_FLOAT_EQ(fast_samples[1].coefficient, 0.4);
-  EXPECT_FLOAT_EQ(fast_samples[2].uv_offset.x, 1.3333333);
-  EXPECT_FLOAT_EQ(fast_samples[2].uv_offset.y, 0);
-  EXPECT_FLOAT_EQ(fast_samples[2].coefficient, 0.3);
+  EXPECT_FLOAT_EQ(blur_info.sample_data[0].x, -1.3333333);
+  EXPECT_FLOAT_EQ(blur_info.sample_data[0].y, 0);
+  EXPECT_FLOAT_EQ(blur_info.sample_data[0].z, 0.3);
```

Comment:
```suggestion
  EXPECT_FLOAT_EQ(GetCoefficient(blur_info.sample_data[0]), 0.3);
```

I think we would be better off not losing the names of values.  It's very confusing to have to remember that `.z` means "get the coefficient".  Can we codify that into some tiny functions?

Same thing for `uv_offset`.

Reply:
Done